src/scss/_projects.scss

```diff
@@ -1,6 +1,9 @@
 .projects {
-    &__wrapper {
-        background-color: $h1-font-color;
+    margin: 150px 0 115px 0;
+    /*надо переводить в % (wh) или em и делать одинаковым для каждой section*/
+    text-align: center;
+    &__block {
+        background-color: #f4f4f4;
         padding: 10% 15%;
```

Comment:
Do you think that you need padding in % here?

Reply:
+